llvm/lib/Target/X86/X86ISelLowering.h

```diff
@@ -789,6 +789,10 @@ namespace llvm {
     // Perform an FP80 add after changing precision control in FPCW.
     STRICT_FP80_ADD,
 
+    // Conditional compare instructions
+    CCMP,
+    CTEST,
+
     // WARNING: Only add nodes here if they are strict FP nodes. Non-memory and
```

Comment:
The comment says you cannot add non strict FP nodes here.

Reply:
Good catch. Done.